src/ecdsa/curves.py

```diff
@@ -166,6 +167,13 @@ def __repr__(self):
     "brainpoolP512r1",
 )
 
+# TODO: Correct the oid and openssl_name
+# TODO: Compose Unit-tests
+Wei25519 = Curve("Wei25519",
+                        ecdsa.curve_wei_25519,
+                        ecdsa.generator_wei_25519,
+                        (1, 3, 101, 110), 
+                        "wei25519")
```

Comment:
if a curve isn't supported in OpenSSL we shouldn't add its `openssl_name`, it's just that we didn't have curves like that before, so the test suite can't handle it correctly—i.e. empty `openssl_name` should be supported in test suite, the fact it's not is a bug